core-processor/src/executor.rs

```diff
@@ -93,9 +71,36 @@ fn make_checks_and_charge_gas_for_pages<'a>(
             return Err(ExecutionErrorReason::InitialMemoryGasExceeded);
         }
 
-        static_pages
+        return Ok(static_pages);
```

Comment:
Comment please

Reply:
Sorry didn't get what there should be commented.

Reply:
In case of init we return here and skip some below gas charging - explain in comment why